docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.install

```diff
@@ -945,3 +945,58 @@ function su_humsci_profile_update_9749(): string {
   }
   return t('domain_registration module was already uninstalled.');
 }
+
+/**
+ * Remove unused image styles and related responsive image styles.
+ */
+function su_humsci_profile_update_9750() {
+  // List of unused image styles to remove.
+  $image_styles_to_remove = [
+    'hero_banner',
+    'hero_large',
+    'hero_no_text_1500px_',
+    'hero_no_text_tall_1500px',
+    'hero_no_text_tall_3000px',
+    'hero_small',
+    'hs_slideshow_large_2000x1500',
+    'hs_slideshow_medium_1500x1125',
+    'hs_slideshow_small_1000x750',
+    'hs_slideshow_xsmall_800x600',
+    'hs_vertical_rectangle_360x430',
+    'large',
+    'postcard_vertical_linked_1057px_',
+    'postcard_vertical_linked_1169px_',
+    'postcard_vertical_linked_465px_',
+    'postcard_vertical_linked_658px_',
+    'postcard_vertical_linked_850px_',
+
+  ];
+
+  // List of responsive image styles that depend on those.
+  $responsive_styles_to_remove = [
+    'hero_image',
+    'portrait_rectangle',
+    'postcard_vertical_linked',
+    'slideshow_responsive',
+  ];
+
+  $entity_type_manager = \Drupal::entityTypeManager();
+
```

Comment:
Could we add some logging and error catching here? No need to log every deletion it can be pretty basic but it's helpful for any debugging. E.g., A count of styles deleted total at the end and a count of any that failed (and then individually log the failure).

Use `\Drupal::logger('su_humsci_profile')` for this and try/catch blocks.

The `su_humsci_profile_update_9743` update in this file has a few good examples of this -- let me know if you have questions.